mslib/msui/topview.py

```diff
@@ -362,7 +359,7 @@ def openTool(self, index):
                                                             mscolab_server_url=self.mscolab_server_url,
                                                             token=self.token)
 
-                self.mainwindow_signal_logout_mscolab.connect(lambda: self.signal_logout_mscolab.emit())
```

Comment:
as discussed in https://github.com/Open-MSS/MSS/pull/2310#commits-pushed-0dd6124 this was the culprit